### ISSUE TITLE
Colors are difficult to read - there are too many items "highlighted"

### ISSUE THREAD

**Issue by borysboyko** [egaga/openvim#17]
![image](https://cloud.githubusercontent.com/assets/970007/11388416/c1752710-92ea-11e5-9a55-bcfa103bba42.png)

As seen on the screenshot, there are several pieces of text "highlighted" with different colors. This makes it rather difficult to track the cursor in the tutorial :(

**Comment by egaga** [comment]
Yes, I thought about that but not sure whether dumping down the amount of highlighted words is the best solution. I need to figure it out if I'll update the style someday. Thanks for the feedback.

**Comment by borysboyko** [comment]
Thanks for the tutorial :)
